### PR TITLE
changed mp3 quality UI to friendlier wording

### PR DESCRIPTION
**Pull request by BigDataStructure**
Resolves: *https://github.com/audacity/audacity/issues/7008*

First ever commit to an open-source project so apologies in advance.
This issue seemed pretty simple so I tried my hand at it.

Chose these words specifically because of FL Studio's naming conventions.

- Changed the Insane and Extreme strings in `ExportMP3.cpp`  to Highest and High, respectively
- Left the backend PRESET_INSANE untouched
- Did the same to every occurrence inside language files
- Did NOT change the words in other languages

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: modules/import-export/mod-mp3/ExportMP3.cpp [hunk 1]
```diff
@@ -181,17 +181,15 @@ static const TranslatableStrings varModeNames {
 };
 */
 static const TranslatableStrings setRateNames {
-   /* i18n-hint: Slightly humorous - as in use an insane precision with MP3.*/
-   XO("Insane, 320 kbps"),
-   XO("Extreme, 220-260 kbps"),
+   XO("Highest, 320 kbps"),
+   XO("High, 220-260 kbps"),
```
Comment (LWinterberg) [inline modules/import-export/mod-mp3/ExportMP3.cpp:185]: I disapprove of calling these "highest" and "high". MP3 becomes transparent (=a listener cannot reliably differentiate between the source and the encoded content) at around 192 kbps, depending on content of course. So for most content, "Standard" actually is the highest quality while giving you the smallest file sizes, and everything above that is just wasteful.

**Review by LWinterberg — CHANGES_REQUESTED** [review]

**Comment by petersampsonaudacity** [comment]
Curiously we have had the existing "Insane" and "Extreme" terminology in the MP3 Export dialog since version 1.3.3, which was released on 18May2007.

And no-one has ever mentioned this before ...

**Comment by petersampsonaudacity** [comment]
@BigDataStructure 

In the Issue that spawned this PR #7008  a discussion ensued on revised wording.

The outcome was tha @LWinterberg  (responsible for Audacity design) favors **Excessive** in place of **Insane**, but **Extreme** should remain as is.  

And I am strongly minded to agree with him.

**Comment by LWinterberg** [comment]
yeah. Also, just as a note: When doing these changes, it's best to just update the source string in the file itself (in this case, ExportMP3.cpp). That way, the translations will get flagged as in need of updating the next time the .po files get regenerated. The way you did it here would cause all translations to still use the "Insane" translations.

**Comment by BigDataStructure** [comment]
Not sure if I took the most accurate and correct steps, but I force-pushed the audacity branch onto my fork. 

New request: #7014

Thanks again.

**Pull request closed without merging**